README.md

```diff
@@ -23,13 +23,37 @@ practices.
 ## Versioning
 The commit hash of the deployed code is considered its version identifier. The commit hash can be retrieved locally via `git rev-parse HEAD`.
 
-## Setting Up
-
-Contile uses Rust, and requires the latest stable iteration. See
-[rustup.rs](https://rustup.rs/) for how to install this application.
-
-Once Rust is installed you can compile using `cargo build`. This will
-create a development release.
+## Setting Up Your Local Dev Environment
+1. Install Rust. See [rustup.rs](https://rustup.rs/) for how to install on your platform.
+2. Compile Contile using `cargo build`. This will create a development release.
+3. Create connection to ADM server:
+    1. Contile requires a connection to the ADM servers. The contract
+    tests include a stub application that can provide ADM-like returns. See the 
+    [adm_settings_test.json](./adm_settings_test.json) file for reference.
+    2. Run the following from the root of the Contile repository to start the ADM stub:
+    ```shell
+        docker run \
+        --env PORT=5000 \
+        --env RESPONSES_DIR=/tmp/partner/ \
+        --env ACCEPTED_MOBILE_FORM_FACTORS=phone,tablet \
+        --env ACCEPTED_DESKTOP_FORM_FACTORS=desktop \
+        -v `pwd`/test-engineering/contract-tests/volumes/partner:/tmp/partner \
+        -p 5000:5000 \
+        mozilla/contile-integration-tests-partner
+    ```
```

Comment:
nit to make this more concise:
```suggestion
3. Start connection a local ADM instance:
    ```shell
        docker run \
        --env PORT=5000 \
        --env RESPONSES_DIR=/tmp/partner/ \
        --env ACCEPTED_MOBILE_FORM_FACTORS=phone,tablet \
        --env ACCEPTED_DESKTOP_FORM_FACTORS=desktop \
        -v `pwd`/test-engineering/contract-tests/volumes/partner:/tmp/partner \
        -p 5000:5000 \
        mozilla/contile-integration-tests-partner
    ```
``